TetrisBot/TetBrain.cs

```diff
@@ -1,13 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
-using System.Runtime.Serialization;
-using System.Runtime.Serialization.Formatters.Binary;
-using System.Text;
-using System.Threading.Tasks;
 using Tetris_Clone;
-using Newtonsoft.Json;
 
```

Comment:
Thanks, forgot to remove a bunch of usings from when I was trying to figure out how to deep copy.